lib/ACFComposer/ResolveConfig.php

```diff
@@ -121,6 +170,19 @@ protected static function forNestedEntities($config, $parentKeys)
         return $config;
     }
 
+    /**
+     * Validates a configuration array based on given required attributes.
+     *
+     * Usually the field key has to be provided for conditional logic to work. Since all keys are generated automatically by this plugin, you can instead provide a 'relative path' to a field by it's name.
+     *
+     * @param array $config Configuration array.
+     * @param array $requiredAttributes Required Attributes.
+     *
+     * @throws Exception if a required attribute is not present
+     * @throws Exception if the `key` attribute is not present
```

Comment:
Everything over line has a full-stop at the end except these two. Is that intentional?